src/User.js

```diff
@@ -7,27 +7,46 @@ class User {
     this.recipesToCook = [];
   };
 
-  addFavoriteRecipe(id) {
-    if (this.favoriteRecipes.includes(id) === false) {
-        this.favoriteRecipes.push(id);  
+  addFavoriteRecipe(recipe) {
+    if (this.favoriteRecipes.includes(recipe) === false) {
+        this.favoriteRecipes.push(recipe);  
     };
   };
 
-  removeFavoriteRecipe(id) {
-    let index = this.favoriteRecipes.indexOf(id);
+  removeFavoriteRecipe(recipe) {
+    let index = this.favoriteRecipes.indexOf(recipe);
     this.favoriteRecipes.splice(index, 1);
   };
 
-  addRecipeToCook(id) {
-    if (this.recipesToCook.includes(id) === false) {
-        this.recipesToCook.push(id);       
-    }
+  addRecipeToCook(recipe) {
+    if (this.recipesToCook.includes(recipe) === false) {
+        this.recipesToCook.push(recipe);       
+    };
   };
 
-  removeRecipeToCook(id) {
-    let index = this.recipesToCook.indexOf(id);
+  removeRecipeToCook(recipe) {
+    let index = this.recipesToCook.indexOf(recipe);
     this.recipesToCook.splice(index, 1); 
-  };
+	};
+
+	filterFavoriteRecipeByTag(tag) {
```

Comment:
Great job using .reduce in the filterFavoriteRecipeByTag and filterRecipesToCookByTag methods instead of both .map and .filter.